spark/src/test/scala/org/apache/comet/exec/CometAggregateSuite.scala

```diff
@@ -537,23 +537,22 @@ class CometAggregateSuite extends CometTestBase with AdaptiveSparkPlanHelper {
               withSQLConf(CometConf.COMET_BATCH_SIZE.key -> batchSize.toString) {
 
                 // Test all combinations of different aggregation & group-by types
-                (1 to 4).foreach { col =>
-                  (1 to 14).foreach { gCol =>
+                (1 to 14).foreach { gCol =>
+                  (1 to 4).foreach { col =>
```

Comment:
Right, this only covers integer types. The other types like float/double, decimal are covered by other tests in the same suite. We could add them here but it might cause explosion in the total test time.